### ISSUE TITLE
Rebrand Site

### ISSUE THREAD

**Issue by joffemd** [govwiki/govwiki.us#18]
Site name and logo are shown here:

https://moqups.com/mdjoffe@gmail.com/AIZuicHG

Limit coverage to just state="CA" ; selection of entities in states other than "CA" can go into a future release.  Our initial grant just covers California.

**Comment by vadim-ivlev** [comment]
Ok

**Comment by vadim-ivlev** [comment]
1. I am going to connect both state and gov_type filters to the map
   control, as the code is the same, and it doesn't make sense leave different
   versions of the same code in the app.
2. Then I will make CA default state and
3. hide state filter for future use.

Is that ok?

**Comment by joffemd** [comment]
Yes, that is okay, but I hope you can also change the site name as per the Moqup.  We can do other cosmetic stuff later.

**Comment by vadim-ivlev** [comment]
Site name is changed.
Coverage is defaulted to state="CA". Have not hidden the Sate filter yet for debugging purposes.
It's interesting to play with it changing the state filter and seeing how dots are moving.)

**Comment by joffemd** [comment]
I agree with not removing the state filter yet.  But can you add the California Policy Center logo: http://californiapolicycenter.org/

**Comment by vadim-ivlev** [comment]
The logo is placed.
Still mot sure about the place, color etc.
I'd like to discuss cosmetic at a later stage.

**Comment by joffemd** [comment]
Yes.  It is good enough for now.
